utils/notificationhubServicePartition.ts

```diff
@@ -0,0 +1,26 @@
+import { Option, some } from "fp-ts/lib/Option";
+import { getConfigOrThrow } from "./config";
+import { ExtendedNotificationHubService } from "./notification";
+
+export function getNHLegacyService(): Option<ExtendedNotificationHubService> {
+  return some(createNH0Service());
+}
+
+export function getNHService(
+  fiscalCodeHash: string
+): Option<ExtendedNotificationHubService> {
+  // tslint:disable-next-line: no-tslint-disable-all
+  // tslint:disable-next-line
+  const fs = fiscalCodeHash;
+
+  return some(createNH0Service());
+}
+
+export function createNH0Service(): ExtendedNotificationHubService {
+  const config = getConfigOrThrow();
```

Comment:
Avoid having side effects inside functions (thrown exceptions are side effects, too).
Viable alternatives:
1. declare `const config = getConfigOrThrow();` at module level, so it's the entire app which won't start if the config isn't good
1. Return `Either<Error, ExtendedNotificationHubService>`

Number 1 is good enough in my opinion, but make your on considerations.

Reply:
Done in [eeae731](https://github.com/pagopa/io-functions-pushnotifications/pull/4/commits/eeae73106a00e5552c744ef0e3771d9715cfd51d)